src/sentry/api/endpoints/organization_issues_count.py

```diff
@@ -22,7 +22,7 @@
 
 
 @region_silo_endpoint
-class OrganizationIssuesCountEndpoint(OrganizationEventsEndpointBase):
+class OrganizationIssuesCountEndpoint(OrganizationEndpoint):
```

Comment:
Does not look like this class needs to extend `OrganizationEventsEndpointBase`
extending `OrganizationEndpoint` instead